_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -1,17 +1,212 @@
+| Relationship   | Recipient       | Third Party |     ✓     | `USER DECISION / NOT ALLOWED` - depending on confidentiality | With this combination the **Sender requests a RelationshipAttribute from a Relationship between the Recipient and a third party. The Attribute must be owned by the Recipient.**<br> Example: A Social Network asks for Facebook privacy settings of a user to get senseful defaults of its own privacy settings. |
+| Relationship   | Third Party     | Third Party |     ✓     | `USER DECISION / NOT ALLOWED` - depending on confidentiality | With this combination the **Sender requests a RelationshipAttribute from a Relationship between the Recipient and a third party which is owned by the third party.**<br> Example: An online shop asks for the Payback Customer Id of a user to book the order on his account.                                     |
+
+### Example of reading an IdentityAttribute
+
+We assume that the Sender wants to read an IdentityAttribute of type [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) of the Recipient. To do this, it inserts the corresponding IdentityAttributeQuery into the `query` property of the [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem), which is contained within the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request) for reading Attributes. In our example, we have chosen to set the value of the `mustBeAccepted` property of the ReadAttributeRequestItem to `true`.
+
+```jsonc
+{
+  "@type": "Request",
+  "items": [
+    {
+      "@type": "ReadAttributeRequestItem",
+      "mustBeAccepted": true,
+      "query": {
+        "@type": "IdentityAttributeQuery",
+        "valueType": "EMailAddress"
+      }
+    }
+  ]
+}
+```
+
+### Example of reading a RelationshipAttribute
+
+We now consider the case that the Sender has an active Relationship established with the Recipient and that the Sender wants to read a RelationshipAttribute of this Relationship. Then the associated [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem), which is contained in the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request) for reading Attributes created by the Sender, must contain an appropriate RelationshipAttributeQuery in its `query` property. For example, if the Sender wants to read a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) of type [ProprietaryString]({% link _docs_integrate/attribute-values.md %}#proprietarystring) that is owned by the Recipient and whose `confidentiality` is `"public"`, the Request could look like this:
```

Comment:
"if the Sender wants to read a RelationshipAttribute" sounds like the RelationshipAttribute already exists and the Sender should be able to read it from its own wallet. Maybe rephrasing it to "if the Sender wants to create a new RelationshipAttribute whose content is set by the Recipient" makes it clearer.